src/index.js

```diff
@@ -74,3 +74,39 @@ deleteBtn.addEventListener('click', () => {
 });
 
 renderTasks();
+// readTask();
+updateTask();
+
+clearCompletedBtn.addEventListener("click", clearCompleted);
+
+function clearCompleted() {
+  const tasks = readTask();
+  const updatedTasks = tasks.filter(task => !task.completed);
+  updatedTasks.forEach((task, index) => {
+    task.id = index; 
+  });
+  updateTask(updatedTasks); 
+  renderTasks(); 
+}
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m2_interactive_list.md#:~:text=Implement%20a%20function%20for%20the%20%22Clear%20all%20completed%22%20button), Please remove all completed tasks from the dom and local storage too.

See this recorded video for better understanding of the problems:
https://screenapp.io/#/shared/8a6130c5-78b9-4508-8919-8e6e825b310a